data/sarthaksarthak9.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "Sarthak Negi",
+  "type": "personal",
+  "bio": "devOps enthusiast , learning webd , open source advocate!",
+  "tags": ["Sarthak", "Open Source", "GitHub", "Community", "Collaboration"],
+  "links": [
+    {
+      "name": "Follow me in twitter and linkedin",
```

It's twitter link here, so shouldn't it be just like `Follow me on Twitter`. 
Rest looks good to me! 

```suggestion
      "name": "Follow me on Twitter",
```

---

I'll do it!!